packages/react/src/avatar-group/AvatarGroup.css.ts

```diff
@@ -0,0 +1,9 @@
+import { recipe } from "../vanilla-extract";
+
+export const avatarGroup = recipe({
+  base: [
+    {
+      display: "flex",
```

Comment:
let's move this sprinkle prop inline to the tsx - since there are no complex styles here

Reply:
Vai, I need the style for styling in avatar.css